addons/account/tests/test_chart_template.py

```diff
@@ -267,3 +268,40 @@ def test_message_to_accountants(self):
             ('partner_ids', 'in', normal_user.partner_id.ids),
             ('body', 'like', f"%{self.tax_template_1.name}%"),  # we look for taxes' name that have been sent in the message's body
         ]), 0)
+
+    def test_update_taxes_coa_installed_via_fp(self):
+        """ When a chart_template is installed through the fiscal position system its taxes should also be updated. """
```

Comment:
Not sure to see how this test is linked to the content of this PR. Was it something broken ? :o

Reply:
It's precisely the point of this PR: when you have taxes installed via the fiscal position system they aren't taken into account by the upgrade script. 
Example: 
- I have a company in Belgium with `l10n_be` and Belgium as fiscal country. 
- I install `l10n_lu`'s taxes via fiscal position system because I also need them. 
=> Before this PR even if we called `update_taxes_from_template(...)` on `l10n_lu`'s chart_template, taxes were not updated.

I will update method name and docstring to make clearer what I test here !